src/assets/environments/environment.json

```diff
@@ -45,5 +45,28 @@
       "redirectSignIn": "https://stage-revalidation.tis.nhs.uk",
       "redirectSignOut": "https://stage-revalidation.tis.nhs.uk"
     }
+  },
+  "alpha": {
+    "production": true,
```

Comment:
Everything else here seems to be stage/preprod, is this correct?

Reply:
> Everything else here seems to be stage/preprod, is this correct?

Yes, that's correct. Alpha is using all the backend services from preprod including cognito. The only real differences is once its authenticated, it returns to alpha FE